src/test/java/uk/gov/dhsc/htbhf/page/GuidancePage.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.dhsc.htbhf.page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.Map;
+
+/**
+ * Page object for the guidance pages of the application
+ */
+public class GuidancePage {
```

Comment:
Would I be right in thinking that this actually represents a collection of `GuidancePages`, not a single `GuidancePage`?

Reply:
Yes, as per the equivalent in the web-ui, this does represent any of the guidance pages as they are all pretty much equivalent apart from path and title for testing.